.github/ISSUE_TEMPLATE/acceptance_test.yml

```diff
@@ -0,0 +1,27 @@
+name: SAS Acceptance Test
+description: Acceptance Testing Procedure for SAS Deliveries
+title: "<SAS Name> <Delivery> SAS <Version> Acceptance Test"
+labels: ["must have"]
+body:
+  - type: textarea
```

Comment:
I think these might need to be changed to markdown types. You can preview how the template will render by viewing the file. ( ... -> View File)

![image](https://github.com/user-attachments/assets/9b1235d7-6dd8-4585-ba84-5c1b606f8cb0)

Reply:
I think I may have misunderstood what is being rendered here and the text area is definitely what we want.

Reply:
Didn't know about the View File feature, but it looks like that preview is rendered differently than how github itself would display the issue form, so it looks like we'll still just have to wait till branch is merged